.idea/weeat.iml

```diff
@@ -48,8 +48,12 @@
     <orderEntry type="library" scope="PROVIDED" name="coffee-script-source (v1.12.2, rbenv: 2.3.1) [gem]" level="application" />
```

Comment:
consider adding IDE/idea files to `.gitignore`

Reply:
:+1: